theme/templates/resource-landing-page/title-section.html

```diff
@@ -274,7 +274,13 @@ <h2 id="resource-title">{{ title }}</h2>
 {% endif %}
 
 {% if is_version_of %}
-    <div class="col-xs-12">
+    {% if user.is_authenticated %}
```

Comment:
This could have negative implications. The way to test is to perform a hard refresh (Command-R) on mac of the browser and then resize the window to see if the changed UI jumps or looks odd at different windows sizes.

Reply:
Struggling to understand how a hard refresh would make a difference on this.  It's a template tag which is processed and removed before being served.  It's also served with in the response to the main request so don't think it's ever cached.